bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/items/ManagedItemProvider.java

```diff
@@ -56,6 +57,58 @@
 
     private Collection<ItemFactory> itemFactories = new CopyOnWriteArrayList<ItemFactory>();
 
+    private final Set<ItemInfo> failedToCreate = new HashSet<>();
+
+    private static class ItemInfo {
+        private final String itemType;
+        private final String itemName;
+
+        public ItemInfo(String itemType, String itemName) {
+            super();
```

Comment:
It is perhaps just my personal meaning, but we could drop the call to the default constructor of the parent.